src/AppBundle/Resources/public/js/app.deck.js

```diff
@@ -476,6 +482,9 @@ deck.get_layout_data = function get_layout_data(options) {
 		if (card.deck_requirements.size){
 			size = card.deck_requirements.size;
 		}
+		if (deck.meta && deck.meta.deck_size_selected){
+			size = parseInt(deck.meta.deck_size_selected, 10);

```

Comment:
Not actually checking if card HAS an option for deck_size_selected, but figure this should never make it into the JSON any other way?